mye2e_suite_test.go

```diff
@@ -0,0 +1,32 @@
+package main_test
+
+import (
+	"testing"
+
+  "k8s.io/kubernetes/test/e2e/framework"
+
+  "k8s.io/kubernetes/test/e2e"
+	_ "k8s.io/kubernetes/test/e2e/apimachinery"
+	_ "k8s.io/kubernetes/test/e2e/apps"
+	_ "k8s.io/kubernetes/test/e2e/auth"
+	_ "k8s.io/kubernetes/test/e2e/autoscaling"
+	_ "k8s.io/kubernetes/test/e2e/common"
+	_ "k8s.io/kubernetes/test/e2e/instrumentation"
+	_ "k8s.io/kubernetes/test/e2e/kubectl"
+	_ "k8s.io/kubernetes/test/e2e/lifecycle"
+	_ "k8s.io/kubernetes/test/e2e/lifecycle/bootstrap"
+	_ "k8s.io/kubernetes/test/e2e/network"
```

Comment:
can't you just import this one and leave out the rest if we only care about networking e2es?